lib/Runtime/Math/WasmMath.inl

```diff
@@ -27,9 +27,7 @@ WasmMath::Eqz(int value)
 template<typename T>
 inline T WasmMath::Copysign(T aLeft, T aRight)
 {
-    //return copysign(aLeft, aRight);
-    //Todo:: fix copysign for ChakraFull
-    return 0;
+    return (T)_copysign(aLeft, aRight);
```

Comment:
I think for float we probably want to do _copysignf, then you don't need to cast